microservices/recordings/src/Service/RawRecordingsGetter.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Service;
+
+use Model\RawRecordingInfo;
+
+class RawRecordingsGetter
+{
+    public function __construct(
+        protected readonly string $rawRecordingsDir
+    ) {
+    }
+
+    /**
+     * @return RawRecordingInfo[]
+     */
+    public function getRawRecordings(): array
+    {
+        $files = [];
+
+        if (!is_dir($this->rawRecordingsDir)) {
+            return $files;
+        }
+
+        $dir = opendir($this->rawRecordingsDir);
+        if (!$dir) {
+            return [];
+        }
+
+        while (false !== ($filename = readdir($dir))) {
+            $absoluteFilename = sprintf(
+                "%s%s",
+                $this->rawRecordingsDir,
+                $filename
+            );
+
```

Comment:
I would add a check to only consider actual files and skip directories, links, and so on. The rawRecordingsDir may contain other files that are not actually Recordings.

Also, considering only files that ends with -mix.wav could be a good idea, because other files in the directory that doesn't match that pattern should be ingored, and not considered as errors.

Reply:
Maybe this is also a good point to validate file format to extract callid from the filename

Reply:
This service is here only keep file system functions separated from the logic. It's fully mocked in testing.